src/components/Home-Page/JobsList/job.tsx

```diff
@@ -57,17 +37,27 @@ const Job: React.FC<JobsProps> = (props): JSX.Element => {
                 </div>
             </div>
             <div className={classes.jobFooter}>
-                <Button className={classes.jobBtn} startIcon={<VisibilityOutlinedIcon className={classes.btnIcon} />}> צפייה  </Button>
-                <Button 
-                    className={`${classes.jobBtn} ${isFavorite && classes.boldText}`}
-                    onClick={handleOnClickSaveButton}
-                    startIcon={getStarIcon()}
+                <Button
+                    className={classes.jobBtn} 
+                    startIcon={<VisibilityOutlinedIcon className={classes.btnIcon} />}
+                    onClick={() => setAreFullDetailsShown(true)}    
                 >
-                    {saveButtonText} 
+                    צפייה  
                 </Button>
+                <SaveJobButton isFavorite={isFavorite} handleClick={handleOnClickSaveButton} />
             </div>
+
+            <JobFullDetails
+                isOpen={areFullDetailsShown}
+                setIsOpen={setAreFullDetailsShown}
+                ad={ad}
+                isFavorite={isFavorite}
+                isRamadView={false}
```

Comment:
Not sure about this, since viewing `jobs` can be done also in Homepage, not only in `personal zone of a candidate`. Be careful!

I understand your idea, a job as in `job proposal` rather than `job progresses/interested in`. Since there are 2 types 🤔 

Not sure: I mean by value `false`

Reply:
this is a MR with half-complete code which i thought would be merged by now, just to make it easier for others to work in the meantime :) I have a lot of fixes to make (the false is here just for now)